R/AbstractGraphReporter.R

```diff
@@ -291,80 +291,86 @@ AbstractGraphReporter <- R6::R6Class(
 
             ## Color Nodes
 
-            # Flag for us to do stuff later
+            # This flag controls whether nodes are colored by categorical
+            # groups or some continuous attribute
             colorByGroup <- FALSE
```

Comment:
@jameslamb you left this old flag here, which means all of the things downstream that depended on it (including the legend) didn't break when you changed to using `colorFieldIsCategorical`.

Reply:
(Also sorry about missing your comment regarding the legend before.)

Reply:
!!!  ah! Ok let me look into it

Reply:
Did you give up on refactoring the plotting? I don't see this in there anymore.

Reply:
oh weird. Yeah I guess I must have! It was a little while ago when I updated this PR so I honestly don't remember. 

I think I decided that just directly changing `DependencyReporter` was a more straightforward way to accomplish this `DependencyReporter`-specific thing